pkg/apis/kops/bastion.go

```diff
@@ -20,4 +20,10 @@ type BastionSpec struct {
 	BastionPublicName string `json:"bastionPublicName,omitempty"`
 	// IdleTimeoutSeconds is the bastion's Loadbalancer idle timeout
 	IdleTimeoutSeconds *int64 `json:"idleTimeoutSeconds,omitempty"`
+
+	LoadBalancer *LoadBalancerSpec `json:"loadBalancer,omitempty"`
+}
+
+type LoadBalancerSpec struct {
```

Comment:
I'm wondering about this type. We probably shouldn't reuse the LoadBalancerAccessSpec since it has fields that only make sense for the k8s API load balancer. It's probably best to have a separate type for this, in which case maybe we name the type something bastion-specific like `BastionLoadBalancerSpec`? That way we can avoid confusion with any other LoadBalancer spec in the kops API. You could keep the json field name as `loadBalancer`.

Reply:
Make sense, let me fix this.